hack/deploy-karmada.sh

```diff
@@ -3,59 +3,84 @@
 set -o errexit
 set -o nounset
 
+# This script deploy karmada control plane to any cluster you want.	REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# This script intended used in following scenarios:	source ${REPO_ROOT}/hack/util.sh
```

Comment:
Do you mean this script depends on utils in `hack/util.sh`?

Reply:
yep